src/vcpkg/export.cpp

```diff
@@ -486,12 +482,7 @@ namespace vcpkg::Export
     {
         const auto cmake_toolchain = prefix / "scripts/buildsystems/vcpkg.cmake";
         const CMakeVariable cmake_variable = CMakeVariable("CMAKE_TOOLCHAIN_FILE", cmake_toolchain.generic_u8string());
-        print2("\n"
-               "To use the exported libraries in CMake projects use:"
-               "\n"
-               "    ",
-               cmake_variable.s,
-               "\n\n");
+        msg::println(msg::format(msgUsingExportedLibs).append_indent().append_raw(cmake_variable.s).append_raw("\n"));
```

Comment:
```
    DECLARE_MESSAGE(CMakeUsingExportedLibs, (), "{value} is a CMake command line switch of the form -DFOO=BAR", "To use exported libraries in CMake projects, add {value} to your CMake command line.");
```

```suggestion
        msg::println(msg::format(msgCMakeUsingExportedLibs, msg::value = cmake_variable.s));
```

I think including the bit in the message makes it more meaningful to localizers.